OnlineStore.Gateway/Program.cs

```diff
@@ -1,34 +1,28 @@
-using Ocelot.DependencyInjection;
-using Ocelot.Middleware;
+using OnlineStore.Gateway.Configurations;
 using Serilog;
 
-var builder = WebApplication.CreateBuilder(args);
+SerilogConfiguration.ConfigureLogging();
 
-builder.Host.UseSerilog((context, configuration) =>
+Log.Information("Starting up");
+
+try
 {
-    configuration.ReadFrom.Configuration(context.Configuration);
-});
+    var builder = WebApplication.CreateBuilder(args);
 
-builder.Configuration.AddJsonFile("ocelot.json", optional: false, reloadOnChange: true);
-builder.Services.AddOcelot(builder.Configuration);
-builder.Services.AddControllers();
-builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
+    builder.Host.UseSerilog();
 
-var app = builder.Build();
+    var app = builder
+        .ConfigureServices()
+        .ConfigurePipeline();
 
-if (app.Environment.IsDevelopment())
+    app.Run();
+}
+catch (Exception ex) when (ex.GetType().Name is not "StopTheHostException" && ex.GetType().Name is not "HostAbortedException")
```

Comment:
Why do u choose to use strings instead of classes with the same names. I mean smth like that: ex is not StopTheHostException.